org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/Decl.java

```diff
@@ -22,6 +22,8 @@
 
 /**
  * Immutable; this declaration binds a list of names to an expression.
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
added a new attribute to declarations, whether a variable/mutable expression is being declared. this is used for the declaration of variable fields (declarations elsewhere cannot be variable).